### DISCUSSION TITLE
Working with VPN/Proxy

### DISCUSSION THREAD

**Discussion by sohamaa** [mashb1t/Fooocus#30] [Ideas]
Can Focus work when we use VPN/Proxy? 

I got this when I turned on them. 
For some locations, it's essential to have it on when we wait for processing ends. 

![image](https://github.com/mashb1t/Fooocus/assets/119129683/3325c003-d535-4546-82dc-f30bd49acf26)

**Comment by mashb1t** [comment]
Yes, but it seems like your VPN doesn't support websocket forwarding / connection upgrades. Please check the technical specs.
